scripts/release-docker.sh

```diff
@@ -57,9 +57,11 @@ bot)
   ;;
 coda-daemon)
   DOCKERFILE_PATH="dockerfiles/Dockerfile-coda-daemon"
+  DOCKER_CONTEXT="dockerfiles/"
```

Comment:
change to dockerfiles/scripts as the dockerfiles themselves don't make any sense inside the image

Reply:
Context doesn't affect what goes in the image (only sets the stage for the daemon to discover/identify filesystem objects to be added). I can see changing to `dockerfiles/scripts` but It's somewhat a practive to set to a root path to allow ops like COPY to indicate relative paths for better clarity.